docusaurus.config.js

```diff
@@ -67,16 +63,28 @@ const config = {
           },
         ],
       },
-        announcementBar: {
-        id: "support_us",
+      announcementBar: {
         content:
-          "Waku is not production ready yet. Join our <a href='https://discord.gg/Nrac59MfSX'>discord community</a> to stay updated.",
+          "While Waku still has progress to make, apps like <a href='/overview/who-is-using-waku'>XMTP and Railgun</a> have already implemented it in production. Join our <a href='https://discord.gg/Nrac59MfSX'>Discord Community</a> to stay updated on our progress.",
```

Comment:
```suggestion
          "While Waku still has progress to make, apps like <a href='/overview/who-is-using-waku'>XMTP and Railgun</a> have already implemented it in production. Join our <a href='https://discord.gg/Nrac59MfSX'>Discord Community</a> to stay updated on our progress.",
```
Let's adjust the message again, needs to be a bit stronger. Maybe something on the line "Using Waku comes with a number of risks and limitations..."